### PR TITLE
Complete Game Class

### PR DESCRIPTION
**Pull request by Shock253**
This should be good, let me know if any of my testing needs work!

### CODE REVIEW

**Comment by javier-aguilar** [comment]
Testing looks solid. I can't think of anything else that is needed. Also, not sure if you meant to leave the commented section at the end of the game test file.

File: test/game_test.rb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      date_time:    "5/16/13",
+      away_team_id: 3,
+      home_team_id: 6,
+      away_goals:   2,
+      home_goals:   3,
+      venue:        "Toyota Stadium",
+      venue_link:   "/api/v1/venues/null"
+    }
+
+    game_data1 = game_data_template.clone
```
Comment (steve-meyers) [inline test/game_test.rb:20]: I'm not familiar with the clone method, but I can see what it is doing here.

File: test/game_test.rb [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+  def test_it_checks_tie
+    assert_equal false, @game1.tie?
+    assert_equal false, @game2.tie?
+    assert_equal true, @game3.tie?
+  end
+
+end
+
+# === DONE ===
```
Comment (steve-meyers) [inline test/game_test.rb:81]: Looks good, we just need to remember to remove these commented lines at some point.

File: test/game_test.rb (hunk 1 above)
Comment (Shock253) [reply test/game_test.rb:20]: it's just copying the hash so I'm not modifying the original

**Pull request merged**